### ISSUE TITLE
Service create wrong response

### ISSUE THREAD

**Issue by Avadon** [docker-php/docker-php#235]
I'm successfully creating a service with following code:

```
$containerSpec = new ContainerSpec();
$containerSpec->setImage('nginx:alpine');

$taskSpec = new TaskSpec();
$taskSpec->setContainerSpec($containerSpec);

$serviceSpec = new ServiceSpec();
$serviceSpecMode = new ServiceSpecMode();
$serviceSpecMode->setGlobal(new GlobalService());
$serviceSpec->setMode($serviceSpecMode);
$serviceSpec->setTaskTemplate($taskSpec);

$result = $this->client->getServiceManager()->create($serviceSpec, []);
```

I checked, service is creating and running after executing this method. But getServiceManager()->create() return null instead of correct service ID.

In Docker\API\Resource\ServiceResource in create() method line 

`$response->getBody()->getContents()`

returns correct json with ID

> '{\"ID\":\"4o2r541o6pdht2rm059nx6se6\"}

but then deserialize() method

`$this->serializer->deserialize((string) $response->getBody(), 'Docker\\API\\Model\\ServiceCreateResponse', 'json')`

turning correct JSON into null

**Comment by mhemrg** [comment]
same issue...

**Comment by joelwurtz** [comment]
will be fixed in 2.0

**Comment by joelwurtz** [comment]
It should be good on 2.0
